code/modules/cooking/machinery/cooking_machines/container.dm

```diff
@@ -148,13 +144,119 @@
 	shortname = "shelf"
 	desc = "Put ingredients in this; designed for use with an oven. Warranty void if used."
 	icon_state = "ovendish"
-	center_of_mass = list("x" = 16,"y" = 12)
 	max_space = 30
-	max_reagents = 120
-	unacidable = TRUE
+	volume = 120
+	appliancetype = OVEN
+
+/obj/item/reagent_containers/cooking_container/skillet
+	name = "skillet"
+	shortname = "skillet"
+	desc = "Chuck ingredients in this to fry something on the stove."
+	icon_state = "skillet"
+	volume = 15
+	force = 15
```

Comment:
What's with all these cooking items being extremely robust, knock them down to like 8

Reply:
```suggestion
	force = 8
```